pkg/proxy/ipvs/util/ipvs_linux.go

```diff
@@ -183,9 +186,15 @@ func (runner *runner) GetRealServers(vs *VirtualServer) ([]*RealServer, error) {
 	if err != nil {
 		return nil, fmt.Errorf("could not convert local virtual server to IPVS service: %w", err)
 	}
+	var dsts []*libipvs.Destination
+
 	runner.mu.Lock()
-	dsts, err := runner.ipvsHandle.GetDestinations(svc)
+	err = retry.OnError(util.MaxAttemptsEINTR, util.ShouldRetryOnEINTR, func() error {
```

Comment:
I would not bother much with this proxy until we have a reported bug, we have more to lose than to win here

Reply:
I had wondered as much. I think bubbling up an error is actually going to be fine, and I assume users may not even notice.